core/src/main/scala/org/apache/spark/internal/config/package.scala

```diff
@@ -357,6 +357,11 @@ package object config {
       .intConf
       .createWithDefault(256)
 
+  private[spark] val HADOOP_OUTPUTCOMMITCOORDINATION_ENABLED =
+    ConfigBuilder("spark.hadoop.outputCommitCoordination.enabled")
+      .booleanConf
```

Comment:
.doc("when enabled, tasks will coordinate with the driver to make sure that, for a certain partition, at most one task attempt can commit.")

Reply:
Done @cloud-fan

Reply:
Done @cloud-fan Thanks!